crates/miden-lib/asm/kernels/transaction/lib/account.masm

```diff
@@ -1260,22 +1261,31 @@ proc.hash_map_key
     # => [HASHED_KEY]
 end
 
-#! Computes the account storage commitment by hashing the storage slots and updates the storage
-#! commitment in memory to the newly computed storage commitment.
+#! Makes the account storage commitment up-to-date.
+#!
+#! Depending on whether the storage has been updated, this procedure does the following:
+#! - If the storage commitment is outdated (account's storage has been changed), this commitment is 
+#!   recomputed by hashing the storage slots. Then this newly computed storage commitment updates 
+#!   the storage commitment in memory.
+#! - If the storage commitment is up-to-date, the the commitment value stored in memory is returned.
```

Comment:
I think this needs to be updated: I don't think this procedure returns anything.